src/native/crt.h

```diff
@@ -175,4 +175,11 @@ JNIEnv *aws_jni_acquire_thread_env(JavaVM *jvm);
  ******************************************************************************/
 void aws_jni_release_thread_env(JavaVM *jvm, JNIEnv *env);
 
+/*******************************************************************************
+ * aws_jni_new_crt_exception_from_error_code - Creates a new jobject from the aws
+ * error code, which is the type of software/amazon/awssdk/crt/CrtRuntimeException.
+ * Reference of the jobject needed to be cleaned up after use.
+ ******************************************************************************/
+jobject aws_jni_new_crt_exception_from_error_code(JNIEnv *env, int error_code);
```

Comment:
OMG how have we gotten this far without already having this function?

Reply:
That's all Dengke.